src/apps/companies/apps/edit-history/client/EditHistoryList.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import DateUtils from 'data-hub-components/dist/utils/DateUtils'
+import { FONT_SIZE } from '@govuk-react/constants'
+import styled from 'styled-components'
+import { GREY_2 } from 'govuk-colours'
+
+import EditHistoryChangeList from './EditHistoryChangeList'
+import { DUNN_AND_BRADSTREET } from '../constants'
```

Comment:
Worth changing the name to more meaningful one, e.g.

```suggestion
import { AUTOMATIC_UPDATE } from '../constants'
```

Reply:
👍